spec/core/ics-026-routing-module/README.md

```diff
@@ -418,10 +418,8 @@ interface ChanOpenInit {
   order: ChannelOrder
   connectionHops: [Identifier]
   portIdentifier: Identifier
-  channelIdentifier: Identifier
   counterpartyPortIdentifier: Identifier
   counterpartyChannelIdentifier: Identifier
-  version: string
```

Comment:
I believe this still needs to exist on the datagram since we pass into the app callback

Reply:
Yes, that's true. My bad! I will fix it now.